### PR TITLE
Zhu Godsie rank selection for GCCA

### LINKED ISSUE #26: Zhu Godsi for GCCA component selection
First step of GCCA takes the SVD of views separately. See Graspy [select_dimension](https://graspy.neurodata.io/_modules/graspy/embed/svd.html#select_dimension). Selects the embedding dimension based on an identified elbo. Thus accounts for variability in noise across views.

### PR DESCRIPTION
**Pull request by rflperry**
Added utils file in multiview.embed with Graspy's ZG rank selection. Updated GCCA tests. Resolves #26

### CODE REVIEW

File: multiview/embed/gcca.py [hunk 1]
```diff
@@ -59,14 +108,7 @@ def _center(self, X):
...
-        self,
-        Xs,
-        fraction_var=0.9,
-        sv_tolerance=None,
-        n_components=None,
-        tall=False,
-    ):
+    def fit(self, Xs):
         """
         Calculates a projection from each view to a latentent space such that
```
Comment (bdpedigo) [inline multiview/embed/gcca.py]: typo

File: multiview/embed/gcca.py [hunk 2]
```diff
@@ -188,62 +226,63 @@ def fit(
...
 
         Returns
         -------
         Xs_transformed : array-like 2D if view_idx not None, otherwise
-            (n_views, n_samples, n_components)
+            (n_views, n_samples, self.n_components)
         """
 
-        return self.fit(Xs, **fit_params).transform(Xs)
+        return self.fit(Xs).transform(Xs)
```
Comment (bdpedigo) [inline multiview/embed/gcca.py]: very small comment, but this could be in base class

File: multiview/embed/utils.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
+from scipy.stats import norm
+import numpy as np
+from scipy.sparse.linalg import svds
+
+
```
Comment (bdpedigo) [inline multiview/embed/utils.py]: fine to copy this stuff, just add something saying where you got it?

Comment (rflperry) [reply multiview/embed/utils.py]: See #46

File: multiview/embed/gcca.py (hunk 1 above)
Comment (rflperry) [reply multiview/embed/gcca.py]: See #45, also related to slack comments you made

File: multiview/embed/gcca.py (hunk 2 above)
Comment (rflperry) [reply multiview/embed/gcca.py]: See #47

**Pull request merged**